ONSdigital/go-ns
# Gracefully shutdown the server in the correct place

### What

Updated server code to gracefully shutdown server after ListenAndServe function has existed

### How to review

- Create a new server in a main() function
- Call ListenAndServe()
- Run and kill the application
- Ensure no errors returned in the logs

### Who can review

Anyone
